### PR TITLE
feat: experimental nuxt-telemetry

### PR DESCRIPTION
**Pull request by atinux**
## Types of changes

Add anonymous telemetry system to better understand how our users uses Nuxt and keep improving Nuxt for the masses.

## Description

https://github.com/nuxt/telemetry

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly. (PR: https://github.com/nuxt/docs/pull/1922)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by pi0 — APPROVED** [review]

File: packages/config/src/options.js [hunk 1]
```diff
@@ -473,5 +474,19 @@ export function getNuxtConfig (_options) {
     staticAssets.versionBase = urlJoin(staticAssets.base, staticAssets.version)
   }
 
+  // Nuxt Telemetry
+  if (
+    options.telemetry !== false &&
+    !options.test &&
+    !destr(process.env.NUXT_TELEMETRY_DISABLED) &&
```
Comment (clarkdo) [inline packages/config/src/options.js:481]: Do we need a new dependecy `destr` here ?

Comment (pi0) [reply packages/config/src/options.js:481]: Dependency is added. It is to check for `true` value not just env existence.

Comment (clarkdo) [reply packages/config/src/options.js:481]: I mean do we need this dependency for just one checking here ?

Comment (pi0) [reply packages/config/src/options.js:481]: No, actually I created it to use in more places including runtimeConfig (convert `"true"` to `true`) :see_no_evil:

**Review by clarkdo — APPROVED** [review]

**Pull request merged**